src/olympia/addons/tests/test_models.py

```diff
@@ -1648,6 +1648,23 @@ def test_new_version_of_approved_addon_should_reset_nomination(self):
         self.check_nomination_reset_with_new_version(addon, nomination)
 
 
+class TestThemeDelete(TestCase):
+
+    def setUp(self):
+        super(TestThemeDelete, self).setUp()
+        self.addon = addon_factory(type=amo.ADDON_THEME)
```

Comment:
I know this is confusing... but what we want here is a "Persona", not a regular addon theme (which are "Complete" Themes). So `amo.ADDON_PERSONA` is the type you want to use here.